### PR TITLE
COMMERCE-1660 Fixed prices visuals

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**

### CODE REVIEW

File: commerce-theme-minium/commerce-theme-minium/src/css/minium/components/price.scss [hunk 1]
```diff
@@ -13,33 +13,31 @@
 
   &-value {
     font-weight: 500;
+    font-size: 20px;
```
Comment (ethanbustad) [inline commerce-theme-minium/commerce-theme-minium/src/css/minium/components/price.scss]: sort please :)

File: commerce-theme-speedwell/commerce-theme-speedwell/src/css/components/_minium_card.scss [hunk 2]
```diff
@@ -199,31 +200,38 @@
...
+        display: none;
       }
 
-      @include specialPricesStyles;
-
-      &-promo {
+      &-value {
+        font-weight: 600;
+        line-height: 27px;
         font-size: 20px;
```
Comment (ethanbustad) [inline commerce-theme-speedwell/commerce-theme-speedwell/src/css/components/_minium_card.scss]: sorting

File: commerce-theme-speedwell/commerce-theme-speedwell/src/css/components/_minium_card.scss [hunk 3]
```diff
@@ -199,31 +200,38 @@
...
+          font-size: 14px;
+          line-height: 19px;
+        }
+
+        @include specialPricesStyles;
+
+        &-discount {
+          margin-left: 10px;
+          font-size: 14px;
+          display: none;
```
Comment (ethanbustad) [inline commerce-theme-speedwell/commerce-theme-speedwell/src/css/components/_minium_card.scss]: sorting

**Review by ethanbustad — APPROVED** [review]

**Comment by gianmarcobrunialti** [comment]
All done @ethanbustad How about proposing to add some postCSS automatic sorter for node-sass / libsass? The SF is definitely not useful here.

**Comment by gianmarcobrunialti** [comment]
ci:test

**Comment by marco-leo** [comment]
I think we have to add sf configuration in package.json files @gianmarcobrunialti

**Comment by marco-leo** [comment]
I think we have to add sf configuration in package.json files @gianmarcobrunialti

**Comment by marco-leo** [comment]
@FabioDiegoMastrorilli @gcmznt  please review

**Comment by ethanbustad** [comment]
^ above result must not have included the PR from Riccardo that fixed those two tests

**Pull request merged**